nailgun/entities.py

```diff
@@ -4305,13 +4370,18 @@ def path(self, which=None):
                 'smart_variables',
                 'module_streams',
                 'disassociate',
+                'traces',
+                'traces/resolve',
```

Comment:
As per API documentation, the trace resolving doesn't comes under any specific host coz the endpoint is `<sat_url>/katello/api/traces/resolve`.

This will fail !

Reply:
TBH the nailgun mirrors the representation of APIDoc but there are multiple PATHs in `Host` class which are not following it unfortunately. Those PATHs shouldnt be appended to Host class but should be separate.
e.g `Host Errata` and even this PR should have separate class as `HostTracer` for host specific tracers.

Reply:
TBH the nailgun mirrors the representation of APIDoc but there are multiple PATHs in `Host` class which are not following it unfortunately. Those PATHs shouldnt be appended to Host class but should be separate.
e.g `Host Errata` and even this PR should have separate class as `HostTracer` for host specific tracers.

Reply:
> This will fail !

Let me prove you wrong: https://github.com/Katello/katello/pull/8945/files#diff-e2385b3bc0001ddfb05e639f7ff3278dR17

Reply:
@ogajduse Thanks for that clarification, now the bug to fix the APIDoc should be raised.

Reply:
There is no bug with the API Doc. I see it generated well. https://<any-sat-6.8-snap17-or-newer>/apidoc/v2/hosts_bulk_actions/resolve_traces.html

Reply:
@ogajduse Ohh it seems the https://<any-sat-6.8-snap17-or-newer>//apidoc page doesnt shows both ! Hence I cant trigger that path !

Anyways now this comment can be mark resolved!

Reply:
I am not sure what you can be doing wrong but the actual API path works for me:
```
curl -sku admin:yourpassword https://<any-sat-6.8-snap17-or-newer>/api/hosts/<host_id>/traces
```

```
curl -sku admin:yourpassword -H "Content-Type:application/json" -H "Accept:application/json" -X PUT -d '{"trace_ids": ["160"]}' https://<any-sat-6.8-snap17-or-newer>/api/hosts/<host_id>/traces/resolve
```

Reply:
![image](https://user-images.githubusercontent.com/20440883/95786596-3402cc80-0cd8-11eb-9556-88034c5053f1.png)
![image](https://user-images.githubusercontent.com/20440883/95786626-4250e880-0cd8-11eb-81c4-8bdbc57414a2.png)

Reply:
Cool! It seems I am looking at some different path. 

I have already marked the command as resolved.